Documentation: New extended tutorial for Liqo. (#787)
liqotech/liqo: docs/pages/gettingstarted/_index.md

weight: 1
---

### Introduction

> Liqo enables the possibility to create simple and straightforward multi-cluster topologies, proposing a seamless multi-cluster model which dynamically allows the pods and services offloading to remote clusters.

Reviewer: ```suggestion
Liqo enables the creation of arbitrary multi-cluster topologies, proposing a seamless multi-cluster model which dynamically allows the pods and services to be offloaded on remote clusters.
```
